src/NzbDrone.Core/Datastore/Migration/140_remove_chown_and_folderchmod_config.cs

```diff
@@ -0,0 +1,17 @@
+using System.Data;
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(140)]
+    public class remove_chown_and_folderchmod_config : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Execute.Sql("DELETE FROM Config WHERE Key = 'folderchmod'");
+            Execute.Sql("DELETE FROM Config WHERE Key = 'chownuser'");
+            Execute.Sql("DELETE FROM Config WHERE KEY = 'chowngroup'");
```

Comment:
Nitpick: 'KEY' -> 'Key'...
Also consider `Key IN ('folderchmod', 'chownuser', 'chowngroup')` turns it into a single query

Reply:
Sorry typo :/